lib/condor.py

```diff
@@ -277,6 +278,7 @@ def record_vault_storer_run(schedd: str, handle: str, outbase: str) -> None:
     # recreate/touch the file
     with open(f"{outbase}/.cvs_{schedd}_{handle}", "w", encoding="utf-8"):
         pass
+    # print("wrote cvs file...")
```

Comment:
Before we merge this, could you please remove the debug print statements?

Reply:
Actually, cleaned them up a bit, and made it a verbose print...